pkg/metrics/metrics.go

```diff
@@ -198,6 +204,22 @@ func NewServerMetrics() *ServerMetrics {
 				},
 				[]string{scheduleLabel},
 			),
+			backupWarningTotal: prometheus.NewCounterVec(
+				prometheus.CounterOpts{
+					Namespace: metricNamespace,
+					Name:      backupWarningTotal,
+					Help:      "Total number of warned backups",
+				},
+				[]string{scheduleLabel},
+			),
+			backupLastStatus: prometheus.NewGaugeVec(
+				prometheus.GaugeOpts{
+					Namespace: metricNamespace,
+					Name:      backupLastStatus,
+					Help:      "Last status of the backup. A value of 1 is success, 0 is failure",
```

Comment:
It's better to rephrase `Last status of the backup` to `Status of schedule's last backup`.

Reply:
My bad.

Reply:
If it goes to `Status of schedule's last backup`, It seems this metrics is only applicable to schedule backup, but it works for one-off backup as well. In some user cases, it could be helpful to use this metrics for one-off backup.